src/content/test/semantics/headers-attribute.tsx

```diff
@@ -19,11 +19,11 @@ export const infoAndExamples = create(({ Markup, Link }) => (
         <h2>How to fix</h2>
         <ul>
             <li>
-                Good: Modify the <Markup.Term>headers</Markup.Term> attribute of the <Markup.Term>{`<td>`}</Markup.Term> element(s) to match
-                the <Markup.Term>id</Markup.Term> attribute of the correct <Markup.Term>{`<th>`}</Markup.Term> element(s), or
+                Good: Modify the <Markup.Code>headers</Markup.Code> attribute of the <Markup.Code>{`<td>`}</Markup.Code> element(s) to match
```

Comment:
Should the td/th references in here be using `<Markup.Tag tagName="whatever" />`?